targets/ar71xx-mikrotik

```diff
@@ -4,8 +4,17 @@ config 'CONFIG_GLUON_SPECIALIZE_KERNEL=y'
 # ath5k cards are commonly used with Mikrotik hardware
 packages 'kmod-ath5k'
```

Comment:
This is certainly not the case for 802.11ac compatible devices anymore. Can you speak to that?

Reply:
This i probably true for older RouterBoards which had miniPCI connectors. As we probably won't support these (old) devices I can remove the line. According to [wireless.wiki.kernel.org](https://wireless.wiki.kernel.org/en/users/drivers/atheros) ath5k only supports a/b/g. I'll remove the line in my next version of the patch.

Reply:
https://github.com/freifunk-gluon/gluon/pull/1620/commits/78386e79c99e5a7b6525a54080afe316ec9061de removes 'kmod-ath5k'